src/WebdriverClassicDriver.php

```diff
@@ -1096,14 +1104,11 @@ private function withWindow(?string $name, callable $callback): void
      */
     private function findElement(
         #[Language('XPath')]
-        string $xpath,
-        ?RemoteWebElement $parent = null
```

Comment:
Why was the `$parent` parameter (and the code, that uses it in this method) removed?

Reply:
We never used that functionality and since it's a private method and is not worth the extra condition and code to handle it at this point, I decided to remove it.

Removing that extra argument also helps a bit with switching to interfaces.

Reply:
OK.